docs/_getting-started/index.md

```diff
@@ -121,56 +135,62 @@ web3.fromWei(eth.getBalance("<ADDRESS_1>"),"ether")
 Getting the balance of an account does not require a signed transaction,
 so Clef does not ask for approval, and Geth returns the value.
 
+Note that this step requires the initial synchronization to end. If you get an error message, return to the 
+console with the network node Geth and wait until it is synchronized. You
+know that your Geth is synchronized when it is only importing a small number of blocks (one or two, usually)
+at a time.
```

Comment:
It also says something like "Switching from fast-sync to full-sync mode"

Reply:
Not with `light` mode, which is what I tell them to use:

```
INFO [06-17|14:57:32.817] Imported new block headers               count=192 elapsed=117.422ms number=4,984,880 hash=ed70f8..1c1dec age=1h26m38s
INFO [06-17|14:57:32.952] Imported new block headers               count=192 elapsed=134.393ms number=4,985,072 hash=88ecb2..fc3937 age=38m38s
INFO [06-17|14:57:35.527] Storing CHT                              section=151 head=de4fb1af7e66fa33ae32ebabcb2148ee63ed585257830a2f9c2632502c3b6bc9 root=643929dfbab652df8fb6485ec035004126176a4bbb0dba50a470361e4d3b9035
INFO [06-17|14:57:35.856] Storing bloom trie                       section=151 head=de4fb1af7e66fa33ae32ebabcb2148ee63ed585257830a2f9c2632502c3b6bc9 root=435e977c17573bc3da68b165ea388120b2d1a862774dc1fbb5e3733e20d595d1 compression=1.000
INFO [06-17|14:57:36.236] Imported new block headers               count=152 elapsed=34.369ms  number=4,985,224 hash=61b4ae..b452f5
INFO [06-17|14:57:36.299] Imported new block headers               count=2   elapsed="685.983µs" number=4,985,226 hash=506bd2..38c9bb
INFO [06-17|14:57:37.179] Looking for peers                        peercount=1 tried=5 static=0
INFO [06-17|14:57:42.834] Imported new block headers               count=1   elapsed="182.25µs"  number=4,985,227 hash=c6f8dd..7c18f7
INFO [06-17|14:57:48.286] Looking for peers                        peercount=1 tried=0 static=0

```

I could try it for snap, but it's a very long process so I'm reluctant to do even a snap sync unless necessary.